boards.txt

```diff
@@ -8397,6 +8270,194 @@ adafruit_metro_esp32s2.menu.EraseFlash.none.upload.erase_cmd=
 adafruit_metro_esp32s2.menu.EraseFlash.all=Enabled
 adafruit_metro_esp32s2.menu.EraseFlash.all.upload.erase_cmd=-e
 
+##############################################################
+# Adafruit Metro ESP32-S3
+
+adafruit_metro_esp32s3.name=Adafruit Metro ESP32-S3
+adafruit_metro_esp32s3.vid.0=0x239A
+adafruit_metro_esp32s3.pid.0=0x8145
+adafruit_metro_esp32s3.vid.1=0x239A
+adafruit_metro_esp32s3.pid.1=0x0145
+adafruit_metro_esp32s3.vid.2=0x239A
+adafruit_metro_esp32s3.pid.2=0x8146
+
+adafruit_metro_esp32s3.bootloader.tool=esptool_py
+adafruit_metro_esp32s3.bootloader.tool.default=esptool_py
+
+adafruit_metro_esp32s3.upload.tool=esptool_py
+adafruit_metro_esp32s3.upload.tool.default=esptool_py
+adafruit_metro_esp32s3.upload.tool.network=esp_ota
+
+adafruit_metro_esp32s3.upload.maximum_size=1310720
+adafruit_metro_esp32s3.upload.maximum_data_size=327680
+adafruit_metro_esp32s3.upload.flags=
+adafruit_metro_esp32s3.upload.extra_flags=
+adafruit_metro_esp32s3.upload.use_1200bps_touch=true
+adafruit_metro_esp32s3.upload.wait_for_upload_port=true
+
+adafruit_metro_esp32s3.serial.disableDTR=false
+adafruit_metro_esp32s3.serial.disableRTS=false
+
+adafruit_metro_esp32s3.build.tarch=xtensa
+adafruit_metro_esp32s3.build.bootloader_addr=0x0
+adafruit_metro_esp32s3.build.target=esp32s3
+adafruit_metro_esp32s3.build.mcu=esp32s3
+adafruit_metro_esp32s3.build.core=esp32
+adafruit_metro_esp32s3.build.variant=adafruit_metro_esp32s3
+adafruit_metro_esp32s3.build.board=METRO_ESP32S3
+
+adafruit_metro_esp32s3.build.usb_mode=0
+adafruit_metro_esp32s3.build.cdc_on_boot=1
+adafruit_metro_esp32s3.build.msc_on_boot=0
+adafruit_metro_esp32s3.build.dfu_on_boot=0
+adafruit_metro_esp32s3.build.f_cpu=240000000L
+adafruit_metro_esp32s3.build.flash_size=16MB
+adafruit_metro_esp32s3.build.flash_freq=80m
+adafruit_metro_esp32s3.build.flash_mode=dio
+adafruit_metro_esp32s3.build.boot=qio
+adafruit_metro_esp32s3.build.partitions=default
+adafruit_metro_esp32s3.build.defines=
+adafruit_metro_esp32s3.build.loop_core=
+adafruit_metro_esp32s3.build.event_core=
+adafruit_metro_esp32s3.build.flash_type=qio
+adafruit_metro_esp32s3.build.psram_type=qspi
```

Comment:
I'm not now 100% sure, but OCTAL PSRAM cannot run in QUAD mode. You should as default set it:
`adafruit_metro_esp32s3.build.psram_type=opi`
`adafruit_metro_esp32s3.build.defines=-DBOARD_HAS_PSRAM`